engine/samples/cars/main.cpp

```diff
@@ -105,20 +145,34 @@ static void move(Query<Write<Car>, Write<Position>, Write<Rotation>> query, Read
         rotation->quat = glm::angleAxis(deltaTime->value * car->angVel, glm::vec3(0.0F, 1.0F, 0.0F)) * rotation->quat;
     }
 }
+/// [Move system]
 
 int main(int argc, char** argv)
 {
     Cubos cubos{argc, argv};
+    /// [Adding the env settings plugin]
     cubos.addPlugin(envSettingsPlugin);
+    /// [Adding the env settings plugin]
```

Comment:
No need to document this, its not directly related to the functionality being showcased.